Libraries/Cordio/wsf/sources/targets/freertos/wsf_trace.c

```diff
@@ -218,7 +218,31 @@ void WsfTrace(const char *pStr, ...)
     }
   }
 }
-#endif
+#if WSF_TRACE_ENABLED_VERBOSE == TRUE
```

Comment:
This will allow the build to pass, but will cause problems when the trace options change. Still not sure which outcome is better. 

**TRACE** options could require a re-build between applications. If the library is built with **TRACE=0** it will be optimized for code size and remove all of the trace messaging. Adjacent applications will still build and link with **TRACE=1**, but the trace messages will not be printed until the library is re-built.

If the library is built with **TRACE=2** and adjacent applications build with **TRACE=0**, the messages will not be printed, but the text will remain in the code space until the library is rebuilt with the appropriate option.